Pipfile

```diff
@@ -13,7 +13,7 @@ django-filter = "==2.1.0"
 django-modeltranslation = "==0.13.1"
 wagtail-modeltranslation = "==0.10.2"
 dj-database-url = "==0.5.0"
-graphene-django = "==2.2.0"
+graphene-django = "==2.6.0"
```

Comment:
note to self mainly, what's new in 2.6? Are there things in the docs that make some API stuff easier/cleaner?

Reply:
nothing major that I noticed, but figured i might as well update the dependency